### DISCUSSION TITLE
How to disable label previews?

### DISCUSSION THREAD

**Discussion by Lrrrence** [latex-lsp/texlab#819] [Q&A] [answered]
I used a key command by accident that has enabled the preview of \label, how can I disable this? E.g.
![tRCQegXtw6](https://user-images.githubusercontent.com/66828803/201989454-19fded6c-61a8-4b6f-9aad-b10c4366c304.png)

**Comment by pfoerster** [comment, marked as answer]
@Lrrrence Which editor are you using? It looks like you enabled the "inlay hints".

**Reply by Lrrrence** [reply]
Thanks a lot! It's VSCode, and I really struggled to search for the right thing with so many options. Disabling Texlab also disabled these hints, hence me thinking it was related.

**Reply by pfoerster** [reply]
I like to use `"editor.inlayHints.enabled": "offUnlessPressed"`. Then you can view the inlay hints by holding `Ctrl + Alt`; otherwise, they are hidden.
